actors.md

```diff
@@ -599,11 +602,6 @@ func CommitSector(sectorID SectorID, commD, commR, commRStar []byte, proof SealP
 		Fatal("not enough collateral")
 	}
 
-	// ensure that the miner cannot commit more sectors than can be proved with a single PoSt
-	if miner.Sectors.Size() >= POST_SECTORS_COUNT {
-		Fatal("too many sectors")
-	}
-
 	// Note: There must exist a unique index in the miner's sector set for each
 	// sector ID. The `faults`, `recovered`, and `done` parameters of the
 	// SubmitPoSt method express indices into this sector set.
```

Comment:
Request: could we pick a consistent capitalisation of `SubmitPo[sS]t`?

Reply:
PoSt is the default, but hard to keep correct everywhere